gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/Cluster.java

```diff
@@ -806,6 +810,7 @@ public Builder minInProcessPerConnection(final int minInProcessPerConnection) {
          * be borrowed once to have multiple requests executed against it.  This number controls the maximum
          * number of requests whereas {@link #maxInProcessPerConnection} controls the times borrowed.
          */
+        @Deprecated
```

Comment:
please add the "@deprecated As of release 3.3.4, not direct replaced" javadoc entries for all of these methods you're deprecating.

Reply:
Fixed. Here and other places.